src/test/java/seedu/address/logic/commands/UntagCommandTest.java

```diff
@@ -123,7 +124,7 @@ public void execute_removeGroupFromKevin() throws CommandException {
                 add(new GroupTag("TA"));
             }};
 
-        //        assertEquals(person.getImmutableGroupTags(), groups);
+        assertEquals(person.getImmutableGroupTags(), groups);
```

Comment:
Take note of the Equivalence Partitions as well. If the person already has the group, what should be the result? If you are tagging two groups and one is already present, how should you deal with the other one? If you have time :) But otherwise, you can merge when ready.